apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -1,5 +1,5 @@
 import { html, TemplateResult } from 'lit';
-import type { StoryObj as Story } from '@storybook/web-components';
+import { type StoryObj as Story } from '@storybook/web-components';
```

Comment:
just curious, what's the difference between both? does it make sense to update the generator to use this intead?

Reply:
I don't actually remember making this change, it was either an automatic change or from a merge update. I _think_ they're both equivalent.

Reply:
cool i will test it and see if it's needed.